From 8e0f043e82e73e5bf19179f2d6a22e0b52186c51 Mon Sep 17 00:00:00 2001
From: "Enrico Weigelt, metux IT consult" <info@metux.net>
Date: Mon, 28 Jul 2025 15:30:03 +0200
Subject: [PATCH] fix missing includes of fb.h

Some source files calling functions from fb.h, so this needs to be
included there.

Signed-off-by: Enrico Weigelt, metux IT consult <info@metux.net>
---
 src/amdgpu_glamor_wrappers.c | 1 +
 src/amdgpu_kms.c             | 3 +++
 2 files changed, 4 insertions(+)

diff --git a/src/amdgpu_glamor_wrappers.c b/src/amdgpu_glamor_wrappers.c
index 9f15141..9e578a2 100644
--- a/src/amdgpu_glamor_wrappers.c
+++ b/src/amdgpu_glamor_wrappers.c
@@ -32,6 +32,9 @@
 #include <config.h>
 #endif
 
+#include <fb.h>
+#include <fbpict.h>
+
 #ifdef USE_GLAMOR
 
 #include "amdgpu_drv.h"
diff --git a/src/amdgpu_kms.c b/src/amdgpu_kms.c
index 03ce0ea..fcda477 100644
--- a/src/amdgpu_kms.c
+++ b/src/amdgpu_kms.c
@@ -30,6 +30,9 @@
 
 #include <errno.h>
 #include <sys/ioctl.h>
+
+#include <fb.h>
+
 /* Driver data structures */
 #include "amdgpu_drv.h"
 #include "amdgpu_bo_helper.h"
